src/hotspot/cpu/aarch64/aarch64_sve_ad.m4

```diff
@@ -335,6 +355,47 @@ instruct storeV_partial(vReg src, vmemA mem, pRegGov pTmp, iRegINoSp tmp1,
   ins_pipe(pipe_slow);
 %}dnl
 
+
+// vector reinterpret
+dnl
+define(`REINTERPRET', `
+instruct reinterpret`'(vReg dst) %{
+  predicate(UseSVE > 0 && n->as_Vector()->length_in_bytes() >= $1 &&
+            n->as_Vector()->length_in_bytes() ==
+            n->in(1)->bottom_type()->is_vect()->length_in_bytes());  // src == dst
+  match(Set dst (VectorReinterpret dst));
+  ins_cost(0);
+  format %{ " # reinterpret $dst" %}
+  ins_encode %{
+    // empty
+  %}
+  ins_pipe(pipe_class_empty);
+%}')dnl
+REINTERPRET(16)
```

Comment:
Can it directly use the instruct itself instead of the macro here?

Reply:
OK.